monocle/worker.py

```diff
@@ -786,9 +786,10 @@ def travel_speed(self, point):
 
                 if (normalized not in SIGHTING_CACHE and
                         normalized not in MYSTERY_CACHE):
-                    if (encounter_conf == 'all'
+                    if ((encounter_conf == 'all'
                             or (encounter_conf == 'some'
-                            and normalized['pokemon_id'] in conf.ENCOUNTER_IDS)):
+                            and normalized['pokemon_id'] in conf.ENCOUNTER_IDS))
+                        and self.player_level != None and self.player_level >= 30):
```

Comment:
No encounter doesn't mean no pokemon...
Encounter is used to get IV, CP and moves